src/fileformats/ocd_file_import.cpp

```diff
@@ -2087,9 +2087,9 @@ void OcdFileImport::import(bool load_symbols_only)
 	Q_ASSERT(buffer.isEmpty());
 	
 	buffer.clear();
-	buffer.append(stream->readAll());
+	buffer.append(device()->readAll());
```

Comment:
I have no experience with mmap. It may work here and now. But for mmap, code will be more complex: We will need to make sure to operate on QFileDevice (for using its map() function), and we will need to make sure to unmap() even when exceptions are thrown. OTOH I wonder if could make sense to retain the whole file in memory, for reuse when writing (to preserve original data). The map files are fairly small in contrast to most raster templates, so this is not too expensive.

However, even while `buffer.append' will avoid unneccessary copying for an empty QByteArea, a simple assignment should be used here.